templates/repo/issue/search.tmpl

```diff
@@ -5,7 +5,7 @@
 		<input type="hidden" name="labels" value="{{.SelectLabels}}"/>
 		<input type="hidden" name="milestone" value="{{$.MilestoneID}}"/>
 		<input type="hidden" name="assignee" value="{{$.AssigneeID}}"/>
-		<input name="q" value="{{.Keyword}}" placeholder="{{.i18n.Tr "explore.search"}}..." autofocus>
+		<input name="q" value="{{.Keyword}}" placeholder="{{.i18n.Tr "explore.search"}}...">
```

Comment:
The same as the above.

Reply:
This subtemplate is used on the issue list where I certainly want to remove it.

Reply:
OK. I mean all the explore pages could keep the autofocus